challenges/linked-list/README.md

```diff
@@ -0,0 +1,18 @@
+# Singly Linked List Implementation
+
+## Challenge
+Create a linked list data a structure class with methods and node class.
+
+## Approach & Efficiency
+Terrible approach. Very inefficient. Took over 2 weeks to complete. 
```

Comment:
this should be an actual description of your approach - it doesn't need to be detailed - just a sentence or two.